src/shared/views/Mapper/CardBody.js

```diff
@@ -78,7 +78,7 @@ class _CardBody extends React.PureComponent<Props, State> {
         this.flatlist = null;
         this.scrollEnabled = !props.tutorial;
         this.tasksPerScreen = undefined;
```

Comment:
what does this line say? why do you set it to `undefined` here. 

don't we expect for the the "normal" project type, that tasks per screen is always 6?

Reply:
I set it to `undefined` as a way to indicate that data hasn't loaded from the backend yet. Once it is loaded, `generateTasks()` fills it in with an array of `number of screens` items, each of which is 6 tasks long.